stardict.cc

```diff
@@ -467,9 +467,9 @@ string StardictDictionary::handleResource( char type, char const * resource, siz
       QString articleText = QString( "<div class=\"sdct_h\">" ) + QString::fromUtf8( resource, size ) + "</div>";
 
       const QString srcRegexpStr(
-          "(<\\s*(?:img|script)\\s[^>]*src\\s*=\\s*[\"']?)(?![^>]*(?:data|https?|ftp):)" );
+          "(<\\s*(?:img|script)\\s[^>]*src\\s*=)(?!\\s*[\"']?(?:data|https?|ftp):)(\\s*[\"']?)" );
       const QString linkRegexpStr(
-          "(<\\s*link\\s[^>]*href\\s*=\\s*[\"']?)(?![^>]*(?:data|https?|ftp):)" );
+          "(<\\s*link\\s[^>]*href\\s*=)(?!\\s*[\"']?(?:data|https?|ftp):)(\\s*[\"']?)" );
```

Comment:
Does the `\\s*[\"']?` part have to be duplicated? Cannot it be captured within the `?!` block?

Reply:
No, inside the `?!` block the group is not being matched.